lib/Cake/Cache/Engine/RedisEngine.php

```diff
@@ -227,4 +227,23 @@ public function __destruct() {
 			$this->_Redis->close();
 		}
 	}
+
+/**
+ * Write data for key into cache if it doesn't exist already. 
+ * If it already exists, it fails and returns false.
+ *
+ * @param string $key Identifier for the data
+ * @param mixed $value Data to be cached
+ * @param int $duration How long to cache the data, in seconds
+ * @return bool True if the data was successfully cached, false on failure
+ * @see https://github.com/phpredis/phpredis#setnx
```

Comment:
I personally perfer to use `@link`for URIs and `@see`for FQSENs.

http://phpdoc.org/docs/latest/references/phpdoc/tags/link.html
http://phpdoc.org/docs/latest/references/phpdoc/tags/see.html

But we haven't defined any rules on that in our coding standards:
http://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html#commenting-code